lib/samples/download_vector_tiles_to_local_cache/download_vector_tiles_to_local_cache.dart

```diff
@@ -207,6 +207,27 @@ class _DownloadVectorTilesToLocalCacheState
       },
     );
 
+    _exportVectorTilesJob?.onStatusChanged.listen(
+      (status) {
+        if (status == JobStatus.succeeded) {
+          setState(() => _progress = 100.0);
+          if (mounted) {
+            ScaffoldMessenger.of(context).showSnackBar(
+              SnackBar(
+                content: const Text('Downloaded vector tiles successfully'),
+                duration: const Duration(seconds: 5),
+                backgroundColor: Theme.of(context).colorScheme.secondary,
+              ),
+            );
+          }
+          // Dismiss the progress indicator.
+          setState(() {
+            _isJobStarted = false;
+          });
```

Comment:
This could be the shorter version:
```dart
setState(() => _isJobStarted = false);
```
Also, if the job fails, the progress indicator will never be dismissed. Maybe this needs to go back where it was or we should check for `JobStatus.failed` here.